### PR TITLE
Streamline documentation

### PR DESCRIPTION
**Pull request by DLu**
Formatting changes and condensing the component text.

### CODE REVIEW

**Comment by DLu** [comment]
Any thoughts? @ryanewei

**Comment by yyu** [comment]
Thank you for the PR.

looking...

File: README.md [hunk 1]
```diff
@@ -1,92 +1,68 @@
...
-In the root directory run pip3 install -e .
+## Setup
+### Docker
+Install docker https://docs.docker.com/install/
 
-How to run:
+### Folder Structure
+ 1. Create a new directory : `mkdir ros_migration_ws/src -p`
+ 2. Initialize a Catkin workspace: `cd ros_migration_ws/src ; catkin_init_workspace`
+ 2.	Fetch the package to be migrated into `ros_migration_ws/src/`
```
Comment (yyu) [inline README.md:14]: TABs should be replaced with spaces

File: README.md [hunk 2]
```diff
@@ -1,92 +1,68 @@
...
 
-In the root directory run pip3 install -e .
+## Setup
+### Docker
+Install docker https://docs.docker.com/install/
 
-How to run:
+### Folder Structure
+ 1. Create a new directory : `mkdir ros_migration_ws/src -p`
+ 2. Initialize a Catkin workspace: `cd ros_migration_ws/src ; catkin_init_workspace`
```
Comment (ryanewel) [inline README.md:13]: Did you find catkin_init_workspace to be necessary? We don't actually build the ros1 package and shouldn't need the root level CMakeLists.txt that it creates.

**Review by ryanewel — COMMENTED** [review]
Mostly looks good and definitely cleans up the instructions. Thanks for contributing!

**Pull request merged**